src/js/components/Notification/Notification.js

```diff
@@ -16,8 +17,56 @@ import { Text } from '../Text';
 
 import { NotificationType } from './propTypes';
 
-const Notification = ({ message, onClose, id, status, title, toast }) => {
-  const autoClose = toast?.autoClose === undefined ? true : toast.autoClose;
+const IconTextContainer = ({ href, onClick, ...rest }) =>
+  href || onClick ? (
+    <Box flex>
```

Comment:
Any reason we can't have `<Box flex />` always be the IconTextContainer? If not, can we add a comment here explaining why?

Reply:
We don't need it for the text to span the available width, so was trying to avoid adding another `div` to the DOM in "non-click" cases. I can add a comment.